chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_v1_to_v2_file
+
+logger = logging.getLogger(__name__)
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        return add_convert_args(parser)
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args = process_args(args)
+        main(args)
+
+def add_convert_args(parser: ArgumentParser) -> ArgumentParser:
+    parser.add_argument(
+        "-i",
+        "--input-path",
+        type=str,
+        required=True,
+        help="Path to a v1 model .pt checkpoint file.",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-path",
+        type=str,
+        help="Path at which the converted model will be saved. If not specified, it defaults to the name of the input file suffixed with '_v2.ckpt' in your current working directory.",
+    )
+    return parser
+
+def process_args(args: Namespace) -> Namespace:
+    args.input_path = Path(args.input_path)
+    args.output_path = Path(args.output_path or Path.cwd() / (args.input_path.stem + "_v2.ckpt"))
```

Comment:
I think we want the default output path to be like so: `input_path = path/to/model.pt` → `output_path = path/to/model_v2.ckpt`. Right now it will write to `$CWD/model_v2.ckpt`